### ISSUE TITLE
Tachiyomi doesn't update library

### ISSUE THREAD

**Issue by Elecktronnick** [tachiyomiorg/tachiyomi#9074]
### Steps to reproduce

When i try to update library from update menu or update a category nothing happens, but updating a manga directly works

https://user-images.githubusercontent.com/70485357/218336069-8817bfc7-8b02-458a-bc91-d578ad0f67d6.mp4

### Expected behavior

Update library

### Actual behavior

Nothing happens, but i can update a manga directly

### Tachiyomi version

Preview r5513

### Android version

Android 13

### Device

OnePlus 9RT

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
https://tachiyomi.org/help/faq/#why-does-global-update-skip-some-entries

**Comment by semiya5** [comment]
Arkon, this is an actual bug. Category updates don't trigger updates. But updates tab does trigger them

**Comment by arkon** [comment]
What's the skip entries set to? It's hard to tell if any of the category's items _should_ be updated.

**Comment by Elecktronnick** [comment]
In my video it's set to only "with completed status"

**Comment by arkon** [comment]
I'll reopen this, but I realistically can't look into it more without a lot more info.

**Comment by semiya5** [comment]
Mine is 8 series, 5 of which I'm caught up on and it seems to be random on when it wants to actually trigger updates. If I find out more, I'll add to it

**Comment by Elecktronnick** [comment]
I reinstalled the app and nothing changed. But then i noticed that there was a restriction  "Only on wifi" at "Automatic updates device restriction"(i use mobile network at moment). 

I removed it and it worked! But it's strange cause i updated the library manually. With "only on unmetered network"(my internet is limited) and "when charging "(i wasn't charging my phone) i update the library as usual
